src/hist/plot.py

```diff
@@ -672,7 +690,7 @@ def _plot_ratiolike(
 
         elif view == "pull":
             pulls: np.typing.NDArray[Any] = (
-                hist_values - compare_values
+                hist_values_width_corrected - compare_values
             ) / hist_values_uncert
```

Comment:
Btw, `hist_values_uncert` seems unbound if `other` is not a callable or string but a histogram. This is not something I introduced in this PR so if I'm not mistaken this might warrant a separate issue.

The question would be whether we just want to use `np.sqrt(self.variances())` or in case `other` also has variances, if we want to statistically combine both somehow :man_shrugging: